schedule/install/offline_spvm_install.yaml

```diff
@@ -28,6 +30,7 @@ schedule:
   - console/system_prepare
   - console/hostname
   - console/force_scheduled_tasks
+  - migration/version_switch_origin_system
```

Comment:
I don't understand why you use this function in spvm, any operations done there?